src/options/options.html

```diff
@@ -24,21 +24,22 @@
       <div id="vocab-tabbed-contents" class="vocab section contents magenta-colored raised bottom-accented clingy">
         <div class="content active" data-content-no="1">
           <div class="note">
-            <span><strong>Note:</strong></span>
-            <ul class="margin-bottom-medium">
-              <li>Changes will take effect after you click the save button at the bottom.</li>
-            </ul>
+            <p class="vocab paragraph inherit-colored"><strong>Note:</strong>
+              <ul class="margin-bottom-medium">
+                <li>Changes will take effect after you click the save button at the bottom.</li>
+              </ul>
+            </p>
           </div>
 
-          <h4>I want something that (use-case)</h4>
+          <h6 class="vocab heading inherit-colored">I want something that (use-case)</h6>
```

Comment:
You can remove `inherit-colored` class heading and paragraph

Reply:
I have made the changes now.